main/2021/WG-07-14.md

```diff
@@ -46,4 +46,139 @@ None.
+
+It takes a “fake” spec and runs it through ReSpec, which creates a valid TR doc. ReSpec has some material that shows editors, etc. It has a config that has to be edited from time to time. It generates the front matter and table of contents (richer that what sphinx generates)
+
+LW: this looks very nice. The TOC looks actually useful.
+
+EP: the TOC is more detailed, let you refer to numbered sections, etc. It’s scriptable, we can run it as a GH action, and can be integrated into W3C’s auto-publish. So if we get our action set up to output the sphinx, we can hook it up.
+To get this running we have to decide to move to first public working draft (starting over after our REC, basically), and then to decide to do automatic publication.
+
+LW: how does that relate to using an evergreen standard?
+
+EP: technically they are orthogonal. Auto-publication just lets you keep it up to date when you iterate on a working draft. Evergreen is that you can stay in CR, and not have to go to REC every couple years, etc. 
+
+LW: we don’t merge stuff too much, it’s mostly like we just did, with a “really done” chunk of work.
+
+EP: yeah when you go to CR you stabilize for 3 months, to update accessibility, internationalization, keep a summary of what’s changed, etc. With evergreen, the patent commitment and review process you did to get to CR, you want to make sure you don’t invalidate the previous patent/accessibility/i18n reviews etc by starting everything over or whatever.
+Once you have auto-publish you can just update the doc with git pushes.
+
+LW: now that we have the tool, that seems quite easy.
+
+EP: yeah if it needed a lot of manual manipulation, it would be slowed down, so this is more realistic. The value of evergreen is less if you need lots of doc work.
+Mathjax has a way to pickle its output, but there are some issues, that might need patching. Are there currently any scripts that change the content when you load the sphinx HTML output? If so, I haven’t caught them. (i did just include all the scripts that were in the old doc). If there are things we want to freeze that were from scripts that run from sphinx, they'd have to be made to work in JSDOM.
+In Firefox you can switch it to MathML and mathjax just does its thing. Do you know what it does when you do that? Is that output something you’d want to pickle to HTML, e.g. for other browsers? Do you know anything about mathML in the other browsers?
+
+DS: the state of mathML in webkit browsers is... complicated.
+
+EP: the mathml WG seems to be reanimated. It seems like maybe the browsers are back on board. It would be nice to be able to just do that once the support is there.
+
+LW: is it possible to just dynamically generate HTML from the mathML if the browser doesn’t support it?
+
+EP: sort of. It’s what the current version does with mathjax. In the future it might be possible to do it AOT. or we could have both a mathML and mathjax version.
```

Comment:
s/we could have both a mathML and mathjax version/we could have both mathML and mathjax markup/

not what i said, but what i wish i'd said, and i think what folks understood.